### PR TITLE
EA-1467 Removed references to client in text, upped version to 1.2.6 and increased minSdk supported version to 26

### PR DESCRIPTION
**Pull request by katekolchier**

### CODE REVIEW

File: bwell-kotlin-android/app/src/main/java/com/bwell/sampleapp/activities/ui/health_journey/HealthJourneyFragment.kt [hunk 1]
```diff
@@ -83,6 +83,7 @@ class HealthJourneyFragment : Fragment() {
         val taskRequest = TasksRequest.Builder()
             .smartSort(true)
             .status(listOf(TaskStatus.READY))
+            //.activityType(listOf("care-need"))
```
Comment (wootencl) [inline bwell-kotlin-android/app/src/main/java/com/bwell/sampleapp/activities/ui/health_journey/HealthJourneyFragment.kt:86]: Should this be removed entirely?

**Review by wfieldx — APPROVED** [review]
Looks good. Maybe address Carter's feedback

File: bwell-kotlin-android/app/src/main/java/com/bwell/sampleapp/activities/ui/health_journey/HealthJourneyFragment.kt (hunk 1 above)
Comment (katekolchier) [reply bwell-kotlin-android/app/src/main/java/com/bwell/sampleapp/activities/ui/health_journey/HealthJourneyFragment.kt:86]: I thought it'd be good to leave in as an example. Every time I want to filter to only Care Needs, I have to go look up the exact string to search on.

Comment (wootencl) [reply bwell-kotlin-android/app/src/main/java/com/bwell/sampleapp/activities/ui/health_journey/HealthJourneyFragment.kt:86]: Makes sense. Merge away

**Review by wootencl — APPROVED** [review]

**Pull request merged**